infra/src/Pos/Infra/Network/DnsDomains.hs

```diff
@@ -1,15 +1,28 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Note: I'll deal with these orphans when I raise my PR for unifying orphans in `infra`.